src/java.base/share/classes/java/text/Format.java

```diff
@@ -185,6 +189,12 @@ public abstract StringBuffer format(Object obj,
                     StringBuffer toAppendTo,
                     FieldPosition pos);
 
+    StringBuilder format(Object obj,
+                         StringBuilder toAppendTo,
```

Comment:
Can we change this type to `StringBuf`, like `StringBuf format(Object obj, StringBuf toAppendTo, FieldPosition pos)`? Then we can remove a few `StringBuilder` overloads by just using the `StringBuf` version.